llpc/context/llpcCompiler.h

```diff
@@ -85,8 +85,8 @@ class GraphicsShaderCacheChecker {
 private:
   Compiler *m_compiler;
   Context *m_context;
-  llvm::Optional<CacheAccessor> m_nonFragmentCacheAccessor;
-  llvm::Optional<CacheAccessor> m_fragmentCacheAccessor;
+  std::optional<CacheAccessor> m_nonFragmentCacheAccessor;
```

Comment:
Note that `#include <optional>` is missing in this header while in the internal porting this is added.

Reply:
Yes, sorry, I added that #include when I was testing the internal version.

Reply:
Never mind. Just need a tiny fix :)